model/items.go

```diff
@@ -47,6 +46,11 @@ type RequestPostOwnersBody struct {
 	Count      int  `json:"count"`
 }
 
+type GetItemResponse struct {
+	IsLiked bool `json:"isLiked"`
+	Item    Item `json:"item"`
+}
```

Comment:
```suggestion
type GetItemResponse struct {
	IsLiked bool `json:"isLiked"`
	Item `json:"item"`
}
```
ですね
レスポンスの形を変えるのはクライアントがバグるのでできるだけ避けたいです

Reply:
直しました